src/Microsoft.AspNetCore.Server.IIS/IISServerOptions.cs

```diff
@@ -22,5 +22,10 @@ public class IISServerOptions
         /// IIS has a non-anonymous authentication enabled, or for back compat with ANCMs that did not provide this information.
         /// </summary>
         internal bool ForwardWindowsAuthentication { get; set; } = true;
+
+        /// <summary>
+        /// Populates the ITLSConnectionFeature
+        /// </summary>
+        public bool ForwardClientCertificate { get; set; } = true;
```

Comment:
Why do we need this?

Reply:
We don't, it was copied from out-of-proc. Remove it.